File: src/jp/lionas/alexa/fundra/FundraStreamHandler.java
package jp.lionas.alexa.fundra;

import com.amazon.ask.Skill;
import com.amazon.ask.SkillStreamHandler;
import com.amazon.ask.Skills;

public class FundraStreamHandler extends SkillStreamHandler {

    private static Skill getSkill() {
        return Skills.standard()
                .addRequestHandlers(
                        new CancelandStopIntentHandler(),
                        new StartIntentHandler(),
                        new AnswerYesIntentHandler(),
                        new AnswerNoIntentHandler(),
                        new HelpIntentHandler(),
                        new LaunchRequestHandler(),
                        new SessionEndedRequestHandler())
                .build();
    }

    public FundraStreamHandler() {
        super(getSkill());
    }
}

File: src/jp/lionas/alexa/fundra/FundraStreamHandler.kt
package jp.lionas.alexa.fundra

import com.amazon.ask.Skill
import com.amazon.ask.SkillStreamHandler
import com.amazon.ask.Skills

class FundraStreamHandler() : SkillStreamHandler(skill) {

    companion object {
        val skill: Skill
            get() = Skills.standard()
                    .addRequestHandlers(
                            CancelAndStopIntentHandler(),
                            StartIntentHandler(),
                            AnswerYesIntentHandler(),
                            AnswerNoIntentHandler(),
                            HelpIntentHandler(),
                            LaunchRequestHandler(),
                            SessionEndedRequestHandler())
                    .build()
    }

}
